tests/all_mode/data/db1.increment.sql

```diff
@@ -25,7 +25,7 @@ update t1 set name = 'gentestxxxxxx' where gen_id = 124;
 delete from t1 where gen_id > 124;
 
 -- test alter database
-alter database all_mode CHARACTER SET = utf8mb4;
+-- alter database all_mode CHARACTER SET = utf8mb4;
```

Comment:
Is it still not support this?

Reply:
This can't be passed because the default CHARACTER is set to latin1. If we revise this line to
```sql
-- alter database all_mode CHARACTER SET = latin1;
```
I think we can pass it.